### DISCUSSION TITLE
How to blur alwyas blur a specific app

### DISCUSSION THREAD

**Discussion by rew1nter** [yshui/picom#1228] [Q&A] [answered]
Is there a way to always blur a specific app despite the inactive/active-opacity rules?
This is my current config https://xgob.in/b1y6w3b3

**Comment by antnesswcm** [comment, marked as answer]
### opacity是透明度，和blur不是同一个东西
我已经让活动窗口和非活动窗口的opacity可以区分了，
### 如果你要为窗口设置固定的opacity只需要使用rulse配置即可，请参照man
也可以看看我的配置
```
{
    match = "class_g = 'st-256color'";
    opacity = 0.3;
    blur-background = false;
    shadow = true;
 },
```
这样st的透明度将保持为0.3
另外，在现在版本的picom中，有许多选项已经弃用，我们最好不要用他
